Sources/Orbit/Support/Layout/IgnoreScreenLayoutHorizontalPaddingModifier.swift

```diff
@@ -21,11 +23,13 @@ struct IgnoreScreenLayoutHorizontalPaddingModifier: ViewModifier {
 
 public extension View {
 
-    /// Reverts any horizontal padding provided by `screenLayout` context in a compact width environment.
+    /// Reverts any horizontal padding provided by `screenLayout` context. Can be optionally limited to specific horizontal size class.
     ///
     /// A typical usage is to mimic the edge-to-edge appearance of the `Card` component.
-    func ignoreScreenLayoutHorizontalPadding() -> some View {
-        modifier(IgnoreScreenLayoutHorizontalPaddingModifier())
+    func ignoreScreenLayoutHorizontalPadding(
+        inHorizontalSizeClasses horizontalSizeClasses: Set<UserInterfaceSizeClass?> = [.compact, .regular]
```

Comment:
What about using `ignoringSizeClass: UserInterfaceSizeClass?` instead of `Set` ? As long as there are only two possible, the `Set` might be overkill for now.

Reply:
Agreed, we can change this later if more size classes appear.